ceph.spec.in

```diff
@@ -493,8 +493,18 @@ BuildArch:      noarch
 Group:          System/Filesystems
 %endif
 Requires:       ceph-mgr = %{_epoch_prefix}%{version}-%{release}
+%if 0%{?fedora} || 0%{?rhel} > 7 || 0%{?suse_version}
+Requires:       python%{_python_buildid}-numpy
+%if 0{without python2}
```

Comment:
Does this also need an extra '%', i.e. `%if 0%{without python2}`?

Also, why not just `Requires: python%{_python_buildid}-scipy` as is done with numpy?

Reply:
`_python_buildid` expands to an empty string if `0%{with python2}`. but we need "2" in this case.

Reply:
Ah, OK.